GHE_main.cpp

```diff
@@ -0,0 +1,237 @@
+//
+//  GHE_main.cpp
+//  
+//
+//  Created by Ryan Davies on 6/14/22.
+//
+
+#include <stdio.h>
+#include <string>
+#include <tuple>
+#include <vector>
+#include <numeric>
+#include <cmath>
+using namespace std;
```

Comment:
I know a lot of code out there on the internet uses this, but I would suggest trying to avoid this where reasonable.  Adding `std::vector` everywhere makes it clear where you got this `vector` thing, and it's very little more code to process.  As you work on larger software projects, that may have their own "vector" or "Vector" classes, it becomes more important to quickly understand where a type is defined.